podpac/core/algorithm/signal.py

```diff
@@ -246,7 +246,13 @@ def _make_kernel(kernel_type, ndim):
             k = np.ones([size] * ndim)
         else:
             args = [float(a) for a in kernel_type.split(",")[2:]]
-            f = getattr(scipy.signal, ktype)
+            if ktype=='gaussian':
```

Comment:
@mpu-creare given the nature of the change, this seemed a little more stable than the try-except but correct me if I'm wrong.

Reply:
Agreed. 

I think you can get rid of the outer `if ktype==gaussian`, the `hasattr` check is good enough. That way, I bet this fix will apply to more windows.

Reply:
Roger that.